aliasMultinomial.lua

```diff
@@ -0,0 +1,13 @@
+local AM = torch.class('torch.AliasMultinomial') 
```

Comment:
this sounds okay. in the Torch world, passing output Tensor is the standard way of doing things. Passing the size is almost never done (as the habit of pre-allocating and re-using buffers is prevalent)